xinference/client/restful/restful_client.py

```diff
@@ -35,6 +35,19 @@
     )
 
 
+def _get_error_string(response: requests.Response) -> str:
+    try:
+        if response.content:
+            return response.json()["detail"]
+    except Exception:
+        pass
+    try:
+        response.raise_for_status()
+    except requests.HTTPError as e:
+        return str(e)
+    return "Unknown error"
```

Comment:
Could we print all stack if has unknown errors.

Reply:
The str(e) may contain the server-side stack.